openslides/core/serializers.py

```diff
@@ -51,12 +51,12 @@ class ProjectorSerializer(ModelSerializer):
     """
     Serializer for core.models.Projector objects.
     """
-    config = JSONSerializerField(write_only=True)
+    config = JSONSerializerField()
```

Comment:
The idea was the config is a write-only field and elements is the read-only counterpart. So the server is able to validate the projector config and clients may see this without any "connection" to the projector itself.

Reply:
I thought the reason was, that you wanted to add other data in the elements-dict that is not part of the config-dict.

The server can still validate the config value in the http request. Am I missing something?